README.md

```diff
@@ -35,10 +35,32 @@ To run this react-native application and start development:
    - `npm install`
 4. Run project:
 
-   - `npm start`
+   - `expo r -c`
```

Comment:
why change this?

Reply:
Had some cache issues with just plain `npm start, and the `-c` fixed it for expo
... also `expo r` is much shorter than `npm start` lmao it was a nitpick

Reply:
Yeah, I just meant clearing the cache prob isn't necessary for first time setup. its doesn't change anything either way.